### PR TITLE
Rename all instances of DHIS 2 or DHIS to DHIS2

### PR DESCRIPTION
**Pull request by jason-p-pickering**

### CODE REVIEW

**Comment by jason-p-pickering** [comment]
@larshelge @ceciliapersson I beleive we have decided on "DHIS2"  (no spaces). Otherwise, let me know.

File: src/docbkx/en/content/analysis/creating_reporting.xml [hunk 1]
```diff
@@ -1195,27 +1195,27 @@ Above the table there are six buttons; five download buttons and one Back button
...
           <listitem>
-            <para>Use the same version of iReport and DHIS 2&apos;s version of Jasper reports. See the About page in DHIS 2 for the Jasper version in use.</para>
+            <para>Use the same version of iReport and DHIS2&apos;s version of Jasper reports. See the About page in DHIS2 for the Jasper version in use.</para>
           </listitem>
           <listitem>
             <para>Use report tables with cross tab dimensions  as your data source for your report designs. This will make it a lot easier to design reports where you need to put specific indicators, periods, or orgunits on columns.</para>
           </listitem>
           <listitem>
-            <para>Learn from others, there are many DHIS 2 report designs for Jasper  on  launchpad, see http://bazaar.launchpad.net/~DHIS 2-devs-core/DHIS 2/trunk/files/head:/resources/</para>
+            <para>Learn from others, there are many DHIS2 report designs for Jasper  on  launchpad, see http://bazaar.launchpad.net/~DHIS2-devs-core/DHIS2/trunk/files/head:/resources/</para>
```
Comment (jason-p-pickering) [inline src/docbkx/en/content/analysis/creating_reporting.xml]: Not sure about this URL. Check it.

File: src/docbkx/en/content/developer/web_api.xml [hunk 2]
```diff
@@ -2069,7 +2069,7 @@ http://server/api/24/constants/ID</screen></para>
   </section>
   <section>
     <title>Metadata export</title>
-    <para>This section explains the new metadata exporter available from DHIS 2.23, the metatada API
+    <para>This section explains the new metadata exporter available from DHIS2.23, the metatada API
```
Comment (jason-p-pickering) [inline src/docbkx/en/content/developer/web_api.xml]: Does this need to be referred to as "DHIS2 2.23"?

**Comment by ceciliapersson** [comment]
I wasn't aware that we had made a decision. I don't mind either or, as long we're consistent. In the e-mail discussion from the 1st of June, there seemed to be a preference for DHIS2.

We need to update www.dhis2.org as well.

File: src/docbkx/en/content/developer/web_api.xml (hunk 2 above)
Comment (ceciliapersson) [inline src/docbkx/en/content/developer/web_api.xml]: I suggest we refer to versions in this way: DHIS2, version 2.23 

In most cases we can omit the name of the software (DHIS2) and only refer to the version (2.23). The section would then be:
"... the new metadata exporter available from version 2.23..." 
The documentation is about DHIS2 so it is not necessary to specify which software the version relates to.

**Comment by jason-p-pickering** [comment]
@ceciliapersson I am not sure there was a final decision either, but my suggestion is that we go ahead and at least be consistent with one or the other. If we need to change it back (once a decision has been made) we can do this rather easily once we have everything consistent.

**Comment by ceciliapersson** [comment]
@jason-p-pickering I agree, let's go ahead with DHIS2.

**Pull request merged**